src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -2,23 +2,31 @@ import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { Router } from '@angular/router';
 import { RouterTestingModule } from '@angular/router/testing';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
 import { provideMockStore } from '@ngrx/store/testing';
-
+import { of } from 'rxjs';
+import { UserService } from 'src/app/services/user/user.service';
 import { HmctsGlobalHeaderComponent } from './hmcts-global-header.component';
 
+
 describe('HmctsGlobalHeaderComponent', () => {
   let nocStoreSpy: jasmine.Spy;
   let component: HmctsGlobalHeaderComponent;
   let fixture: ComponentFixture<HmctsGlobalHeaderComponent>;
   let mockRouter: jasmine.SpyObj<Router>;
 
+  const changesMock = {items: { currentValue: 'a', previousValue: 'b', firstChange: false, isFirstChange: () => false}};
```

Comment:
@ronaldmansveld - hmm... spacing issue. Not sure what to do with this one: looks like some `{` have a space after them, but all `}` have none... maybe get rid of spaces?

Reply:
Lint didn't raise it :P But I'll update

Reply:
Thanks, @ronaldmansveld - it looks much nicer now :)